Makefile

```diff
@@ -109,6 +109,10 @@ build-amd64: build-op-amd64 build-fb-amd64 build-fd-amd64
 build-op-amd64:
 	docker build --platform=linux/amd64 -f cmd/fluent-manager/Dockerfile . -t ${FO_IMG}
 
+# Build Verrazzano specific amd64 Fluent Operator container image
+vz-build-op-amd64:
+	docker build --build-arg BASE_IMAGE=${BASE_IMAGE} --platform=linux/amd64 -f cmd/fluent-manager/Dockerfile . -t ${FO_IMG}
```

Comment:
Are we using --platform arg? It's a nitpick if we aren't.

Reply:
In Jenkins, we are using build-op-amd64, I just added build-arg on top it in vz-build-op-amd64

Reply:
let me know, if something need to be fixed. can be do in following PR